packages/core/useScroll/index.ts

```diff
@@ -114,7 +114,7 @@ export function useScroll(
     directions.right = scrollLeft > x.value
     arrivedState.left = scrollLeft <= 0 + (offset.left || 0)
     arrivedState.right
-          = scrollLeft + eventTarget.clientWidth >= eventTarget.scrollWidth - (offset.right || 0)
+      = scrollLeft + eventTarget.clientWidth >= eventTarget.scrollWidth - (offset.right || 0) - 1
```

Comment:
Can we make the magic `1` as an internal const and reference to the MDN docs?

Reply:
Absolutely, good idea -- I pushed https://github.com/vueuse/vueuse/commit/638ceebcc01d531b1e72aa228ee40e66d05e247c. Thanks!